README.md

```diff
@@ -19,9 +19,8 @@ Open Enterprise is a collection of Aragon apps that enable organizations to cura
 
 ## Development Setup
 
-Node.js LTS or greater required.
-
-- Note: @aragon/cli and truffle npm deps are automatically installed when bootstrapping.
+* Node.js LTS or greater required. You can use a tool like [`asdf`](https://asdf-vm.com/) or [`nvm`](https://github.com/nvm-sh/nvm) to manage versions of node.
```

Comment:
That seems too obvious, and maybe adding `engine` field to `package.json` would help since we are not supporting every node version from LTS right now, add least we didn't test them all, we are in fact using 10.x branches in places like CI, so this statement seems not really accurate.
Said that I like the suggestions about the node versions managers, I didn't know about `asdf` myself, so I liked this message... I let to you the final decision on how to make up this based on my comment @chadoh or just keeping it as is, if you consider that.

Reply:
Oh, gotcha, yeah, that makes sense. I was keeping the comment that had been there somewhat intact, "Node.js LTS or greater required" – I would have preferred to not change it at all, for now. Adding `engine` to `package.json` is on my list of things to do; maybe we can improve this line and add that in a follow-up PR?